Xamarin.Forms.Core/Items/CarouselView.cs

```diff
@@ -161,6 +171,12 @@ public LinearItemsLayout ItemsLayout
 		[EditorBrowsable(EditorBrowsableState.Never)]
 		public bool IsScrolling { get; set; }
 
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public bool IsInitialized = false;
```

Comment:
name should be more explicit. it should also be changed to a property or a method, as it's value is set from outside of this type

Reply:
Moved to a method  a new name